index.html

```diff
@@ -796,6 +796,47 @@ <h2><dfn>SystemAudioPreferenceEnum</dfn></h2>
             </tbody>
           </table>
         </section>
+        <section>
+          <h2><dfn>DynamicSourceOptionPreferenceEnum</dfn></h2>
+          <p>
+            Describes whether an application invoking {{MediaDevices/getDisplayMedia()}}
+            would like the user agent to offer the user the option to dynamically change
+            the source [=display surface=] during the capture.
+          </p>
+          <pre class="idl">
+            enum DynamicSourceOptionPreferenceEnum {
+              "include",
+              "exclude"
```

Comment:
I'd be happy to change this if Youenn and Jan-Ivar agree.

Reply:
Agree that "Allow" and "Deny" is better than "include"/"exclude".

Reply:
Actually, I like "include" and "exclude" here since it's consistent with recent enums we've added around surface inclusion/exclusion. This is slightly different since it's a dynamic feature to switch surfaces later, but seems similar in that it has to do with what surfaces the user can choose.

Those other enums were also hints, so being consistent in naming with them would be nice, and might help readers intuitively understand they're all hints